### DISCUSSION TITLE
Symfony 3.4 upgrade to 4.4

### DISCUSSION THREAD

**Discussion by StaffNowa** [symfony/symfony#42867] [Q&A] [answered]
A very good source for understanding what we need to do
1. https://symfonycasts.com/screencast/symfony-3.3
2. https://symfonycasts.com/screencast/symfony4-upgrade

**Comment by ahundiak** [comment, marked as answer]
Are you asking a question or just providing some links?

**Reply by StaffNowa** [reply]
Provide links ;)

**Reply by chapterjason** [reply]
@StaffNowa In the Q&A category?

**Reply by StaffNowa** [reply]
@chapterjason I can only post into this category 😳

**Reply by chapterjason** [reply]
@StaffNowa I don't know the reason why there is only this category, but I think it is cause it is a beta in first place and like in the announcement its just a kind of testing phase for the symfony community.
